dvc/api/__init__.py

```diff
@@ -1,9 +1,6 @@
-from .data import (  # noqa, pylint: disable=redefined-builtin
-    get_url,
-    open,
-    read,
-)
+from dvc.fs.dvc import _DvcFileSystem as DvcFileSystem
+
+from .data import open  # pylint: disable=redefined-builtin
+from .data import get_url, read
 from .experiments import make_checkpoint
 from .params import params_show
-
-__all__ = ["get_url", "make_checkpoint", "open", "params_show", "read"]
```

Comment:
We don't need `__all__` as we are selectively reimporting stuffs.

Reply:
That does change the behaviour of `from dvc.api import *` though. Without `__all__`, the submodules are exposed.

Reply:
Ahh, right. Thank you for catching that. 🤦🏼